src/libs/ajax/User.ts

```diff
@@ -88,6 +88,14 @@ export const makeSetUserProfileRequest = (terraUserProfile: TerraUserProfile): S
   };
 };
 
+/**
+ * Orchestration's /register/profile endpoint returns profile attributes as an
+ * array of { key, value } objects. This converts that array into single object.
+ */
+export const kvArrayToObject = (kvArray: { key: string; value: any }[] | undefined): Record<string, any> => {
+  return Object.fromEntries((kvArray ?? []).map(({ key, value }) => [key, value]));
```

Comment:
SonarCloud suggested switching from `|| []` to `?? []`.

https://sonarcloud.io/organizations/broad-databiosphere/rules?open=typescript%3AS6606&rule_key=typescript%3AS6606